spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'spec_helper'
+
+describe UserSignup do
+  describe '#signup' do
+    after { ActionMailer::Base.deliveries.clear }
+
+    it 'returns self' do
+      stub_out_stripe_wrapper
+      signup_handle = UserSignup.new(Fabricate.build(:user))
+      response = signup_handle.signup('fake_stripe_token')
+      expect(response).to eq(signup_handle)
+    end
+
+    context 'with valid user personal info and valid card' do
+      before { stub_out_stripe_wrapper }
+      let(:attributes){ Fabricate.attributes_for(:user) }
+      let(:signup_handle){ UserSignup.new(User.new(attributes)) }
+
+      it 'charges users credit card' do
+        expect(StripeWrapper::Charge).to receive(:create)
+        signup_handle.signup('fake_sripe_token')

```

Comment:
You have a typo in these specs `fake_sripe_token`. Copy and paste can bite you, be careful.